SimplenoteWidgets/Models/Note+CoreDataClass.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Note+CoreDataClass.swift
+//  
+//
+//  Created by Lantean on 6/30/21.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(Note)
+public class Note: SPManagedObject {
+
+}
```

Comment:
These were autogenerated classes (by Xcode). Let's please merge both the `Note+CoreDataClass` and `Note+CoreDataProperties` into a single file.

**Plus** please also:

- Drop the dummy comments
- Add a note mentioning this class is there for us to avoid linking Simperium / SPManagedObject, in the widget target